### ISSUE TITLE
Add nan handling strategy for run_vizier.py

### ISSUE THREAD

**Issue by klei22** [ReaLLMASIC/nanoGPT#193]
Maybe we can first find the largest possible validation loss and save this to the ckpt.pt file and best val loss files?

The default 10^9 will be overindexed by the optimization algorithm, maybe we can have the vizier try to skip results that nan before sending a valid value?
